internal/pkg/template/templates/cicd/partials/role-policy-document.yml

```diff
@@ -0,0 +1,77 @@
+Version: '2012-10-17'
```

Comment:
Is this policy document the same (is there any new thing we added here)?

Reply:
It's the same as before.